.ci/travis/travis.sh

```diff
@@ -253,6 +266,10 @@ no-error-test-sbe)
     "s/'com.puppycrawl.tools:checkstyle:.*'/'com.puppycrawl.tools:checkstyle:$CS_POM_VERSION'/" \
     build.gradle
   ./gradlew build --stacktrace
+  cd ..
+  removeFolderWithProtectedFiles simple-binary-encoding
+  cd ..
+  validateTempDirectoryEmpty
```

Comment:
For some reason I feel that this should be done ones, at post script actions, the same as we do for git status

Reply:
See https://github.com/checkstyle/checkstyle/issues/7892#issuecomment-618977743

Reply:
this does not explain this ...
Travis has https://github.com/checkstyle/checkstyle/blob/11baaf25874bcdc1144f8575cf605de2adcd1903/.travis.yml#L358-L361

if you put this function here ... it will be good.

Reply:
It is done.